bosh-director/lib/bosh/director/jobs/ssh.rb

```diff
@@ -36,10 +36,11 @@ def perform
             logger.info("ssh #{@command} '#{instance.job}/#{instance.uuid}'")
             result = agent.ssh(@command, @params)
             if target.ids_provided?
-              result["id"] = instance.uuid
+              result["index"] = result["id"] = instance.uuid
```

Comment:
Hmm, this would end up changing the data format of `index` which I don't think is intentional (maybe some clients require integers). Rather than having a conditional for `target.ids_provided?`, let's just do...

```
result["id"] = instance.id
result["index"] = instance.index
```

Old CLI will look at `index` and have the same info as normal and new CLI will correctly show the uuid.